pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -445,6 +446,7 @@ func realScheduleMonitor(c *Cluster, mon *monConfig) (SchedulingResult, error) {
 		_, err = c.context.Clientset.CoreV1().PersistentVolumeClaims(c.Namespace).Create(pvc)
 		if err == nil {
 			pvcName = pvc.Name
+			d.Labels["pvc_name"] = pvcName
```

Comment:
To implement this, can we add it to the `getLabels()` method and pass an argument for the pvc name? Then the label will be set everywhere needed.

Reply:
Now  I am using the getLabels method, so inserted the labels in all the places needed. I prefered not to add parameters to this function. Because is used only in the spec.go file , and because is easy to set the labels in mon.go when is required

Reply:
It makes sense not to change `getLabels()` if this label is only applied for the mons.

Reply:
and what do you prefer to do @leseb?

Reply:
If @BlaineEXE and @travisn are asking for it then please go ahead and address their request.